services/galley/src/Galley/Intra/Client.hs

```diff
@@ -182,7 +182,12 @@ getLocalMLSClients lusr ss =
             "clients",
             toByteString' (tUnqualified lusr)
           ]
-        . queryItem "sig_scheme" (toByteString' (signatureSchemeName ss))
+        . queryItem
+          "ciphersuite"
+          ( "0x"
+              <> toByteString'
+                (Hex (cipherSuiteNumber (tagCipherSuite suite)))
```

Comment:
Use `instance ToHttpApiData CipherSuite` for consistency

Reply:
Fixed